modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -56,9 +56,11 @@ machines.
 If you disable simultaneous multithreading, ensure that your capacity planning
 accounts for the dramatically decreased machine performance.
 ====
-<4> This parameter controls the number of workers that the cluster deploys for
-you, so you must set the number of workers to `0` in this file. Remember that you
-must manually deploy at least one compute, or worker, machine in the cluster.
+<4> This parameter controls the number of workers that the cluster creates
```

Comment:
@djuran, is this clearer?

Reply:
How about. But please check if the below text is correct. I'm really new to this topic...

With the bare-metal scenario, the hardware is outside the control of the openshift controller and this parameter should be set to 0. 
With that said, the installer will still deploy worker nodes on the hardware using the worker ingition file.

Reply:
No, you have to manually deploy the worker nodes by using the Ignition config that you generated by using the installation program. The cluster recognizes the workers that you deploy instead of deploying them in any way.